src/test/java/org/ahocorasick/interval/IntervalTest.java

```diff
@@ -54,4 +54,14 @@ public void comparable() {
         assertEquals(4, it.next().getStart());
     }
 
+    @Test
+    public void checkToString() {
```

Comment:
Let's migrate the naming schemes for the unit test methods to follow Roy Osherove's recommendation: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html

With one minor modification to adhere to Google's Java style guide conventions: prefix the method name with `test_`.